change/@uifabric-charting-1027db92-a05d-490c-bb5c-874afb13b7d9.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fix wrong narration when legend selected",
+  "packageName": "@uifabric/charting",
```

Comment:
From what I remember, doesn't this change depend on the change where you added active/highlighted legend states.

Reply:
Actually, it was the other way around. #24563 depended on this change